readme.md

```diff
@@ -1,6 +1,9 @@
 # Bitbucket Server API Wrapper
 A simple wrapper for the Atlassian's Bitbucket Server / Bitbucket Datacenter (formerly Stash) REST API, written in Python.
 
+[![Test](https://github.com/JoeStanleySEL/bitbucketserver/actions/workflows/test-and-coverage.yml/badge.svg?branch=master)](https://github.com/JoeStanleySEL/bitbucketserver/actions/workflows/test-and-coverage.yml)
```

Comment:
These links should be to this repo; can we just change the paths to our org and they will work once the scripts run?

Reply:
Absolutely!

I'd only used reference to my personal repo to provide some form of demonstration of how they might look:
![image](https://user-images.githubusercontent.com/108963606/199848052-44f14e04-e686-4b85-9bf0-6c184df6a102.png)

Reply:
Resolving conversation because this change has been made.